src/App.tsx

```diff
@@ -0,0 +1,139 @@
+    let custoPedagio = pedagio * pedagioValor;
+    let total = custoCombustivel + custoPedagio;
+
+    setInfo({
+      combustivel: custoCombustivel,
+      pedagio: custoPedagio,
+      total: total,
+    });
+  }
+
+  function formatarMoeda(valor: number) {
+    let valorFormatado = valor.toLocaleString('pt-br',
+      {
+        style: 'currency',
+        currency: 'BRL'
+      }
+    )
+
+    return valorFormatado
+  }
+
+  return (
+    <>
+      <main className='container'>
+        <img 
+          className='logoImg'
+          src={logoImg} 
+          alt='Logotipo de um carro no posto de combustível com um relógio em cima escrito TRAVEL CALCULATOR' 
+        />
+        <form className='form' onSubmit={calcular}>
+          <label>
+            Distância (km):
+            <input 
+              className='input' 
+              placeholder='100' 
+              type='number' 
+              min={0} 
+              step={0.01} 
+              required 
+              value={distancia}
+              onChange={ (e) => setDistancia(Number(e.target.value)) }
+            />
+          </label>
+          <label>
+            Consumo médio do veículo (km/l):
+            <input 
+              className='input' 
+              placeholder='40' 
+              type='number' 
+              min={0} 
+              step={0.01}
+              required 
+              value={consumo} 
+              onChange={ (e) => setConsumo(Number(e.target.value)) }
+            />
+          </label>
+          <label>
+            Preço do combustível (R$):
+            <input 
+              className='input' 
+              placeholder='6,20' 
+              type='number' 
+              min={0} 
+              step={0.01} 
+              required 
+              value={combustivel} 
+              onChange={ (e) => setCombustivel(Number(e.target.value)) }
+            />
+          </label>
+          <label>
+            Quantidade de pedágios:
+            <input 
+              className='input' 
+              placeholder='2' 
+              type='number' 
+              min={0} 
+              step={1} 
+              required 
+              value={pedagio}
+              onChange={ (e) => setPedagio(Number(e.target.value)) }
+            />
+          </label>
+          {pedagio > 0 && (
+            <label>
+              Valor de cada pedágio (R$):
+              <input 
+                className='input' 
+                placeholder='1,50' 
+                type='number' 
+                min={0} 
+                step={0.01} 
+                required 
+                value={pedagioValor} 
+                onChange={ (e) => setPedagioValor(Number(e.target.value)) }
+              />
+            </label>
+          )}
+          <button type='submit' className='buttonSubmit'>
+            Calcular custo total
+          </button>
+        </form>
+        {info && Object.keys(info).length > 0 && (
```

Comment:
Voce pode simplificar a condicional apenas verificando se tem um total.

```suggestion
        {info?.total && (
```

Reply:
Atualizado